analytic_engine/src/setup.rs

```diff
@@ -270,22 +264,34 @@ fn open_storage(
                         path: sst_path.to_string_lossy().into_owned(),
                     })?;
                 let store = LocalFileSystem::new_with_prefix(sst_path).context(OpenObjectStore)?;
-                Ok(Arc::new(store) as _)
+                Arc::new(store) as _
             }
-            StorageOptions::Aliyun(aliyun_opts) => Ok(Arc::new(AliyunOSS::new(
+            ObjectStoreOptions::Aliyun(aliyun_opts) => Arc::new(AliyunOSS::new(
                 aliyun_opts.key_id,
                 aliyun_opts.key_secret,
                 aliyun_opts.endpoint,
                 aliyun_opts.bucket,
-            )) as _),
-            StorageOptions::Cache(cache_opts) => {
+            )) as _,
+            ObjectStoreOptions::Cache(cache_opts) => {
                 let local_store = open_storage(*cache_opts.local_store).await?;
                 let remote_store = open_storage(*cache_opts.remote_store).await?;
                 let store = CachedStore::init(local_store, remote_store, cache_opts.cache_opts)
                     .await
                     .context(OpenObjectStore)?;
-                Ok(Arc::new(store) as _)
+                Arc::new(store) as _
             }
+        };
+
+        if opts.mem_cache_capacity.as_bytes() == 0 {
```

Comment:
The method name `as_bytes` is confused...

Reply:
Agree, `as_byte` may better match as method like `as_mb()`

Reply:
https://github.com/CeresDB/ceresdb/issues/428